pkg/hintrunner/zero/hintcode.go

```diff
@@ -85,6 +85,6 @@ const (
 
 	// ------ Other hints related code ------
 	allocSegmentCode string = "memory[ap] = segments.add()"
-	vmEnterScopeCode string = "vm_enter_scope()"
+	memcpyEnterScope string = "vm_enter_scope({'n': ids.len})"
```

Comment:
nitpick: rename to memcpyEnterScopeCode 😉

Reply:
fixed!